File: unity-component/Assets/Scripts/Models/SettingsModel.cs
using System.Collections.Generic;
using JetBrains.Annotations;

namespace Models
{
    [System.Serializable]
    public class SettingsModel
    {
        public float predictionScoreThreshold = 0.75f;

        public List<ModelEndpoint> modelEndpoints = new List<ModelEndpoint>();

        [CanBeNull] public ModelEndpoint activeEndpoint;
    }
}

File: native-app/native-app/Sources/Models/SettingsModel.swift
//
//  SettingsModel.swift
//  native-app
//
//  Created by Michael Pavkovic on 6/22/20.
//  Copyright © 2020 Unity Technologies. All rights reserved.
//

import Foundation

struct SettingsModel: Codable {
    
    var predictionScoreThreshold: Float
    var modelEndpoints: [ModelEndpoint]
    var activeEndpoint: ModelEndpoint?
    
}
